### DISCUSSION TITLE
✅ 🔗 Ability to set spacing for pagination in loop carousel widget

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21697] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team @avivu, @nicholaszein,

As a small but useful feature, it would be great if you could add the ability to set the spacing for paging in the loop carousel widget and even nested carousel (in the future).

![pagination](https://user-images.githubusercontent.com/88034902/226211854-486246df-2f3a-4d0f-9a9e-5d0808f7ef59.png)

Currently, this distance cannot be controlled.

### Describe the solution you'd like

New option to control spacing between pagination Dots.

**Comment by nicholaszein** [comment]
Hey @Michaelgimii!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
